src/runtimes/node/bundlers/nft/es_modules.ts

```diff
@@ -49,4 +30,78 @@ const patchESMPackage = async (path: string, fsCache: FsCache) => {
   return JSON.stringify(patchedPackageJson)
 }
 
-export { getPatchedESMPackages }
+const shouldTranspile = (path: string, cache: Map<string, boolean>, reasons: NodeFileTraceReasons): boolean => {
+  if (cache.has(path)) {
+    return cache.get(path) as boolean
```

Comment:
```suggestion
    return cache.get(path)!
```

teeny tiny nit, `!` is a better fit

Reply:
(didn't check this locally, but I assume that `cache.get` returns `boolean | undefined`)